results/cmd/api/main.go

```diff
@@ -81,3 +81,23 @@ func (s *server) CreateTaskRun(ctx context.Context, req *pb.CreateTaskRunRequest
 	}
 	return taskrunFromClient, nil
 }
+
+// Get TaskRun from database and send it to watcher
+func (s *server) GetTaskRun(ctx context.Context, req *pb.GetTaskRunRequest) (*pb.TaskRun, error) {
+	database := s.db
+	rows, err := database.Query("SELECT taskrunlog FROM taskrun WHERE uid = " + req.GetUid())
+	if err != nil {
+		log.Fatalf("failed to query on database: %v", err)
+	}
+	var taskrunblob []byte
+	taskrun := &pb.TaskRun{}
+	for rows.Next() {
```

Comment:
We should log a warning if this returns multiple rows for some reason.

Reply:
done